.uberenv_config.json

```diff
@@ -5,8 +5,9 @@
 "package_source_dir" : "../..",
 "force_commandline_prefix" : false,
 "spack_url": "https://github.com/spack/spack",
-"spack_commit": "e0e6a0199ef4db473ea0bd2e924f3e96fa10297a",
+"spack_commit": "1a080b9a00a7952dd636d9eba7137e61ae4af4b6",
 "spack_configs_path": "scripts/spack/configs",
 "spack_packages_path": "scripts/spack/packages",
-"spack_activate" : {}
+"spack_activate" : {},
+"use_clingo": true
```

Comment:
Please prefix this with "spack_" like all the other spack options.  Also is "use_clingo" descriptive enough without having to understand the internals of spack? spack_use_new_concretizer?